flow/src/commonMain/kotlin/org/onflow/flow/websocket/FlowWebSocketMessage.kt

```diff
@@ -0,0 +1,296 @@
+    @SerialName("transaction_index") val transactionIndex: Int,
+    @SerialName("event_index") val eventIndex: Int,
+    val payload: JsonElement,
+    @SerialName("block_id") val blockId: String,
+    @SerialName("block_height") val blockHeight: Long,
+    @SerialName("block_timestamp") val blockTimestamp: String
+)
+
+// --- Account Status Event ---
+
+@Serializable
+data class AccountStatusEventResponse(
+    @SerialName("subscription_id")
+    override val subscriptionId: String,
+    val topic: String?,
+    val payload: AccountStatusPayload
+) : FlowWebSocketMessage()
+
+@Serializable
+data class AccountStatusPayload(
+    val address: String,
+    val balance: String,
+    val code: List<String> = emptyList(),
+    val keys: List<AccountPublicKey> = emptyList(),
+    val contracts: Map<String, String> = emptyMap(),
+    @SerialName("account_created") val accountCreated: Boolean,
+    @SerialName("block_id") val blockId: String,
+    @SerialName("block_height") val blockHeight: Long,
+    @SerialName("block_timestamp") val blockTimestamp: String
+)
+
+@Serializable
+data class AccountPublicKey(
+    val index: Int,
+    @SerialName("public_key") val publicKey: String,
+    @SerialName("sign_algo") val signAlgo: Int,
+    @SerialName("hash_algo") val hashAlgo: Int,
+    val weight: Int,
+    @SerialName("sequence_number") val sequenceNumber: Int,
+    @SerialName("revoked") val revoked: Boolean
+)
+
+// --- Transaction Status Event ---
+
+@Serializable
+data class TransactionStatusEventResponse(
+    @SerialName("subscription_id")
+    override val subscriptionId: String,
+    val topic: String?,
+    val payload: TransactionStatusPayload
+) : FlowWebSocketMessage()
+
+@Serializable
+data class TransactionStatusPayload(
+    @SerialName("transaction_id") val transactionId: String,
+    val status: TransactionStatus,
+    @SerialName("block_id") val blockId: String,
+    @SerialName("block_height") val blockHeight: Long,
+    @SerialName("block_timestamp") val blockTimestamp: String
+)
+
+@Serializable
+enum class TransactionStatus {
+    @SerialName("UNKNOWN")
+    UNKNOWN,
+
+    @SerialName("PENDING")
+    PENDING,
+
+    @SerialName("FINALIZED")
+    FINALIZED,
+
+    @SerialName("EXECUTED")
+    EXECUTED,
+
+    @SerialName("SEALED")
+    SEALED,
+
+    @SerialName("EXPIRED")
+    EXPIRED
+}
+
+// --- Send Transaction Status Event ---
+
+@Serializable
+data class SendTransactionStatusEventResponse(
+    @SerialName("subscription_id")
+    override val subscriptionId: String,
+    val topic: String?,
+    val payload: SendTransactionStatusPayload
+) : FlowWebSocketMessage()
+
+@Serializable
+data class SendTransactionStatusPayload(
+    @SerialName("transaction_id") val transactionId: String,
+    val status: SendTransactionStatus,
+    @SerialName("block_id") val blockId: String? = null,
+    @SerialName("block_height") val blockHeight: Long? = null,
+    @SerialName("block_timestamp") val blockTimestamp: String? = null
```

Comment:
If we can decode with existing model, then we should avoid introduce new data model
https://github.com/onflow/flow-kmm/blob/c908c86b9fa03d6b26646b2ce5fde68e54cb4abb/flow/src/commonMain/kotlin/org/onflow/flow/models/BlockEvents.kt#L17-L30